src/thunderbots/software/test/primitive/directwheels_primitive.cpp

```diff
@@ -0,0 +1,143 @@
+/**
+ * This file contains the unit tests for the MoveSpinPrimitive class
+ */
+
+#include "ai/primitive/directwheels_primitive.h"
+
+#include <gtest/gtest.h>
+#include <string.h>
+
+TEST(DirectWheelsPrimTest, construct_with_no_params_test)
+{
+    const std::string directwheels_prim_name = "DirectWheels Primitive";
+
+    DirectWheelsPrimitive directwheels_prim = DirectWheelsPrimitive(0, 0, 0, 0, 0, 0.0);
+
+    EXPECT_EQ(int(), directwheels_prim.getRobotId());
```

Comment:
I've discussed this before with @EvanMorcom, I do not like the usage of `int()` here. Yes it almost always defaults to 0, but it is implementation defined and it is not immediately obvious what number we're actually comparing to.